Colours_And_Fonts_style_v1.css

```diff
@@ -128,31 +194,55 @@ ul li:hover ul {
 
 .box1 {
   width: 1990px;
-  height: 20px;
-  background-color: #BC1327;
+  height: 10px;
+  background-color: #A2091B;
 }
 
 .box2 {
 	width: 1990px;
-	height: 20px;
-	background-color: #ffdd00;
+	height: 10px;
+	background-color: #FBCA01;

```

Comment:
I changed box2 from a lighter dark yellow to a lighter yellow so it matches the school colours.
I also changed the height of the box from 20px to 10px as it was too big.